test/src/test/java/hudson/security/csrf/DefaultCrumbIssuerTest.java

```diff
@@ -178,6 +181,7 @@ public class DefaultCrumbIssuerTest {
     @Issue("SECURITY-626")
     @WithTimeout(300)
     public void crumbOnlyValidForOneSession() throws Exception {
+        assumeTrue(runTestSometimes(NEVER_FAILING_TEST));
```

Comment:
Please no.

Reply:
Thanks for the comment and thanks for the pointer to a specific case where a technique like this caused serious problems.  It is good for me to know that this is not a workable technique.

Is there any technique you can envision that would be acceptable to reduce the frequency of running the tests?  I'm fine if there is not, but am curious if you can see any path that would allow us to run these tests less frequently while still running them sometimes.

Reply:
> Is there any technique you can envision that would be acceptable to reduce the frequency of running the tests? 

Skip on master specifically when running on ci.j.io? PRs still run them (even if unrelated), and cert.ci still runs.

Reply:
Thanks for the clarification.  I'll implement that.